manim/animation/animation.py

```diff
@@ -1,50 +1,17 @@
 from copy import deepcopy
 
-import attr
 import numpy as np
-import typing
 
 from ..mobject.mobject import Mobject
 from ..utils.config_ops import digest_config
 from ..utils.rate_functions import smooth
-from ..utils.dataclasses import dclass
-
-if typing.TYPE_CHECKING:
-    from ..scene.scene import Scene
 
 
 DEFAULT_ANIMATION_RUN_TIME = 1.0
 DEFAULT_ANIMATION_LAG_RATIO = 0
 
-@dclass
```

Comment:
Why revert everything we've done on `Animation` though?

Reply:
Yeah sorry I forgot to mention it. I removed it for the moment to test the current version with test_geometry. If I had kept the commit about animation (which is a progress of course) it would require more changes before being able to test it. I thought I could get the animation version from master and then perhaps cherry-pick later what was already done.